template/src/lib.rs

```diff
@@ -13,7 +13,11 @@ use uuid::Uuid;
 pub use askama::Template;
 
 #[derive(Debug, Clone, Template, TypedBuilder)]
-#[template(path = "Containerfile.j2", escape = "none", whitespace = "minimize")]
+#[template(
+    path = "Containerfile.vanilla.j2",
+    escape = "none",
+    whitespace = "minimize"
+)]
 pub struct ContainerFileTemplate<'a> {
```

Comment:
Rather than changing the existing template struct, I would suggest making a brand new template struct.

Reply:
I agree. I just did it like this for testing purposes, and because I kind of wish(ed) that there would be some way to dynamically change the template file without duplicating the struct.

Reply:
Solved here: [622f895](https://github.com/blue-build/cli/pull/206/commits/622f89599ffafc4c9d459539f6b257737ee7a7ed)